tests/test_straight_skeleton_2.py

```diff
@@ -0,0 +1,16 @@
+from compas_cgal.straight_skeleton_2 import create_interior_straight_skeleton
```

Comment:
perhaps it would be worth adding a few tests of "known"/"predictable" skeletons?